sw/device/silicon_creator/manuf/extensions/BUILD.bazel

```diff
@@ -28,6 +28,7 @@ cc_library(
         "@//sw/device/lib/testing/test_framework:status",
         "@//sw/device/lib/testing/test_framework:ujson_ottf",
         "@//sw/device/silicon_creator/lib/cert",
+        "@//sw/device/silicon_creator/manuf/lib:flash_info_fields",
```

Comment:
Looks like you included the wrong dep here. You should be including `@//sw/device/lib/dif:flash_ctrl`: https://github.com/lowRISC/opentitan/blob/34b8fc33e36b3ed2a32e6cd50b451a47f16b202d/sw/device/lib/dif/BUILD#L324

Same below.